client/ayon_core/tools/publisher/widgets/report_page.py

```diff
@@ -1459,7 +1459,7 @@ class CrashWidget(QtWidgets.QWidget):
     def __init__(self, controller, parent):
         super(CrashWidget, self).__init__(parent)
 
-        main_label = QtWidgets.QLabel("This is not your fault", self)
+        main_label = QtWidgets.QLabel("Error Occurred!", self)
```

Comment:
I wonder if `Error Occurred!` is too unfriendly/scary? I recall that being a huge design decision as to why once it was chosen to report with `This is not your fault`. So may be one for @mkolar to comment on?

Reply:
Tbh, I don't know what should be the message. 
I only changed it to open a discussion about it.